tools/express/express_wrapper.xml

```diff
@@ -1,7 +1,7 @@
-<tool id="express" name="eXpress" version="1.1.1">
+<tool id="express" name="eXpress" version="1.5.1">
 	<description>Quantify the abundances of a set of target sequences from sampled subsequences</description>
 	<requirements>
-		<requirement type="package" version="1.1.1">eXpress</requirement>
+		<requirement type="package" version="1.5.1">eXpress</requirement>
 	</requirements>
 	<command>
 		express --no-update-check
```

Comment:
```suggestion
    #if $bamOrSamFile.ext == "qname_sorted.bam"
        ln -s `$bamOrSamFile` hits.sorted
    #else
        samtools sort -n '$bamOrSamFile' hits.sorted
    #end if
    express --no-update-check
```

Plus: replace `$bamOrSamFile` on the last line of the command block with `hits.sorted` (btw. all input files need to be single quoted).

Maybe in the else branch one could check with `samtools view -H ... | grep -q "@HD\t.*\tSO:unsorted"` if the sam / bam file is sorted before calling sort again.